chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -457,14 +457,15 @@ bool ParseJSON(base::StringPiece json,
       { "name": "ropsten-infura.brave.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
 
       // Rewards
-      { "name": "ads-serve.brave.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
```

Comment:
Removed pinning for old host

Reply:
are we not using this for any other hosts?

Reply:
This host is no longer used, thanks